src/pokemon_debug.c

```diff
@@ -623,6 +656,107 @@ void BattleLoadOpponentMonSpriteGfxCustom(u16 species, bool8 isFemale, bool8 isS
     LoadPalette(gDecompressionBuffer, 0x80 + battlerId * 16, 0x20);
 }
 
+//Battle background functions
+struct BattleBackground
+{
+    const void *tileset;
+    const void *tilemap;
+    const void *entryTileset;
+    const void *entryTilemap;
+    const void *palette;
+};
+extern const struct BattleBackground sBattleTerrainTable[];
+#define MAP_BATTLE_SCENE_NORMAL       0
+#define MAP_BATTLE_SCENE_GYM          1
+#define MAP_BATTLE_SCENE_MAGMA        2
+#define MAP_BATTLE_SCENE_AQUA         3
+#define MAP_BATTLE_SCENE_SIDNEY       4
+#define MAP_BATTLE_SCENE_PHOEBE       5
+#define MAP_BATTLE_SCENE_GLACIA       6
+#define MAP_BATTLE_SCENE_DRAKE        7
+#define MAP_BATTLE_SCENE_FRONTIER     8
+#define MAP_BATTLE_SCENE_LEADER       9
+#define MAP_BATTLE_SCENE_WALLACE      10
+#define MAP_BATTLE_SCENE_GROUDON      11
+#define MAP_BATTLE_SCENE_KYOGRE       12
+#define MAP_BATTLE_SCENE_RAYQUAZA     13
```

Comment:
Not sure how I feel about these constants, or the ones near the top of the file.

They should all be put inside an `include/constants/pokemon_debug.h` file imo.

Reply:
followed your suggestion and added a `include/constants/pokemon_debug.h` file for defines